Mathlib/Analysis/InnerProductSpace/Calculus.lean

```diff
@@ -245,6 +245,38 @@ theorem DifferentiableAt.norm (hf : DifferentiableAt ℝ f x) (h0 : f x ≠ 0) :
   ((contDiffAt_norm 𝕜 h0).differentiableAt le_rfl).comp x hf
 #align differentiable_at.norm DifferentiableAt.norm
 
+theorem not_differentiableAt_abs_zero : ¬ DifferentiableAt ℝ (fun x : ℝ ↦ abs x) 0 := by
+  rw [DifferentiableAt]
+  push_neg
+  intro f
+  simp [HasFDerivAt, HasFDerivAtFilter, Asymptotics.isLittleO_iff]
```

Comment:
this looks like a non-terminal simp, please replace it with the output of `simp?`